x-pack/test/api_integration/services/slo.ts

```diff
@@ -21,39 +21,92 @@ type FetchHistoricalSummaryParams = t.OutputOf<
 >;
 
 export function SloApiProvider({ getService }: FtrProviderContext) {
-  const supertest = getService('supertest');
+  const supertest = getService('supertestWithoutAuth');
   const esClient = getService('es');
+  const security = getService('security');
 
   return {
+    async createUser() {
+      const username = 'slo_editor';
+      const roleName = 'slo_editor';
+      try {
+        await security.user.delete(username);
+        await security.role.delete(roleName);
+      } catch (error) {
+        const status = error.response.status;
+        if (status !== 404) {
+          throw error;
+        }
+      }
+      const password = 'changeme';
+
+      await security.role.create(roleName, {
+        elasticsearch: {
+          indices: [
+            {
+              names: ['.slo-*'],
+              privileges: TOTAL_INDEX_PRIVILEGE_SET_EDITOR,
+            },
+          ],
+        },
+      });
+
+      await security.user.create(username, {
+        password,
+        roles: [roleName, 'editor'],
+      });
+    },
     async create(params: CreateSLOInput) {
       const slo = await supertest
         .post('/api/observability/slos')
         .set('kbn-xsrf', 'true')
+        .auth('slo_editor', 'changeme')
         .send(params)
         .expect(200);
 
-      const { id } = slo.body;
+      return slo;
+    },
+    async reset(id: string) {
+      const response = supertest
+        .post(`/api/observability/slos/${id}/_reset`)
+        .auth('slo_editor', 'changeme')
+        .set('kbn-xsrf', 'true')
+        .send()
+        .expect(200);
 
-      const reqBody = [{ id: `slo-${id}-1` }, { id: `slo-summary-${id}-1` }];
-      await supertest
-        .post(`/internal/transform/schedule_now_transforms`)
```

Comment:
@dominiqueclarke We don't need to use schedule_now_transforms anymore?

Reply:
I believe we added the schedule now to the actual implementation logic, so we don't need it in the test anymore.